### DISCUSSION TITLE
Headless mode for @mentions?

### DISCUSSION THREAD

**Discussion by excid3** [josefarias/hotwire_combobox#113] [Ideas]
I've used Zurb foundation for @mentions for years but it's went unmaintained for a while now. It seems like Hotwire Combobox could possibly support this functionality in a headless mode, but I'm not sure the complexities of doing this. This has been a core missing feature of ActionText that I think could be nicely solved.

Curious what you think about this idea @josefarias?

**Comment by josefarias** [comment]
Yes! This has been on my mind from the very beginning but I didn't design around it.

So it's a "yes" from me philosophically,and a "probably?" practically.

Next up on my radar is wrapping up multiselect (probably this week), then grouped options, and third headless. At least for now.

Of course if someone wants to take a stab at it they're more than welcome to. Probably good to list out the requirements here and a rough implementation plan to avoid too much rework.

**Reply by excid3** [reply]
I meant to ask, do you have a rough outline of steps to implement this if I happened to find some free time?

**Reply by josefarias** [reply]
Not yet I'm afraid! But I'll update here if I find time to think it through.
